app/frontend.py

```
@@ -1,16 +1,37 @@
 # app/frontend.py
 import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)